src/package.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <package xmlns="http://www.mendix.com/package/1.0/">
-	<clientModule name="formatstringplus" version="0.1" xmlns="http://www.mendix.com/clientModule/1.0/">
+<!-- Custom upgraded to Mendix 7, based on version 0.1-->
+	<clientModule name="formatstringplus" version="1.0.1" xmlns="http://www.mendix.com/clientModule/1.0/">

```

Comment:
version number should be next major. (so 1.0.0) not major and patch

Reply:
Done